packages/main/src/DatePicker.js

```diff
@@ -549,6 +533,10 @@ class DatePicker extends UI5Element {
 		};
 	}
 
+	get type() {
+		return InputType.Text;
```

Comment:
if this never changes, just use it in the template IMO

Reply:
As it is not used anywhere else in Datepicker.js, it is treeshaked and can't be used in the template.